docker-compose.yaml

```diff
@@ -5,17 +5,31 @@ volumes:
 services:
   mbtileserver:
     image: headway_tileserver
+    build: ./tileserver
     environment:
       PORT: 8000
   photon:
     image: headway_photon
+    build: ./geocoder/photon
   nominatim:
     image: headway_nominatim
+    build: ./geocoder/nominatim
   graphhopper:
     image: headway_graphhopper
+    build: ./graphhopper
     volumes:
     - "headway_graphhopper_vol:/graph_volume"
   nginx:
     image: headway_nginx
+    build: ./web
+    environment:
+      HEADWAY_BASE_URL: http://maps.example.com
```

Comment:
nit: here and in the dockerfile, default to `http://localhost:8080` maybe? seems ideal for development and eventually we'll probably want a kubernetes config for prod anyway.

Reply:
Fixed. Also renamed it to `HEADWAY_PUBLIC_URL` to make the distinction more clear